loops-and-arrays.js

```diff
@@ -57,5 +67,12 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 11 is 1331'
 // ]
 
+console.log(squares)
+
+for (let i = 0; i < squares.length; i++) {
+    console.log(squares[i]*squares[i]*squares[i])
```

Comment:
Wist je dat je dit kon doen?
```suggestion
    console.log(squares[i]**3)
```

Reply:
Ah cool. Als ik het zie denk ik, ja natuurlijk, logischer dit. Dank voor de tip.